data/json/professions.json

```diff
@@ -774,6 +774,40 @@
       "female": [ "panties", "bra", "stockings", "dress_shoes", "maid_dress", "maid_hat" ]
     }
   },
+  {
+    "type": "profession",
+    "ident": "captive",
+    "name": { "male": "Captive", "female": "Captive" },
+    "description": "You were folowing a road at night trying to get away from the horrors of the city when you heard a voice calling out in the dark.  As you stepped away to investigate you suddenly felt a seering pain in your head and blacked out.  You just woke up in this place... Are you even on earth anymore?",
+    "points": 0,
```

Comment:
I think starting with absolutely nothing should probably earn you points.

Reply:
1 point like Shower Victim (or maybe 2) should be fine.